### ISSUE TITLE
0 stories tested?

### ISSUE THREAD

**Issue by tleunen** [storybooks/storyshots#36]
Trying to test `storyshots` inside a project but it gives me 0 stories tested.

The way I'm running storybook is like that: `start-storybook -p 9001 -s ./.storybook/public`
And storyshots is just 'storyshots'.

Am I missing anything?

**Comment by arunoda** [comment]
It's hard to tell like this.
Are you using require.context?

It's better we can have something to try out locally.

**Comment by tleunen** [comment]
Yes sorry, forgot that this project was using the context... (Unfortunately I can't share the project).

Here's the line:

```
const req = require.context('../src', true, /__stories__\/(.*)\.story\.jsx$/);
```

**Comment by arunoda** [comment]
@roonyh Hey, checkout our require.context implementation satisfy this.
If not, let's do a fix.

**Comment by tleunen** [comment]
My regex is based on a path, but storyshots tests it on the filename only
See https://github.com/kadirahq/storyshots/blob/master/src/require_context.js#L10

**Comment by roonyh** [comment]
@tleunen Spot on! Submitted a fix at #37. Will publish soon.

**Comment by roonyh** [comment]
@tleunen Published v1.1.5 with the fix.
